dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -7574,8 +7574,8 @@ BEGIN
     8692 "Săgeată spre dreapta cu cerc mic"
     8693 "Săgeată în jos lângă săgeată în sus"
     8694 "Trei săgeți spre dreapta"
-    8695 "Săgeată spre stânga cu tăietură verticală"
-    8696 "Săgeată spre dreapta cu tăietură verticală"
+    8695 "Săgeată spre stânga tăiată verticală"
+    8696 "Săgeată spre dreapta tăiată verticală"
```

Comment:
To me, this change looks a bit hard to understand, especially when you preferred „cu tăietură” before (see resource № 2098 & 7661). Please take care of consistency.